### DISCUSSION TITLE
Branch 1.5 (vs. changelog generation)

### DISCUSSION THREAD

**Discussion by famod** [quarkiverse/quarkus-cxf#616] [General]
Hi @ppalaga & @shumonsharif,

I can't seem to find a discussion about that new 1.5 branch (main is now on 1.6).
I just wanted to let you know that auto-generating the changelog (when creating a GH release) is based on merged PRs so it doesn't generate anything right now for 1.5.7.
I'll pick the relevant PRs from the 1.6 changelog, but this mode of operation is not ideal.

I don't have a quick solution to offer. Maybe backports should be done in a PR instead?

WDYT? Also cc @gastaldi.

Cheers,
Falko

**Comment by gastaldi** [comment]
@famod Yes, backports should be preferably done in a PR. That will ensure that CI is run for these changes

**Comment by ppalaga** [comment]
Thanks for the heads up @famod. I have no problem sticking with PRs for backports.

**Reply by famod** [reply]
@ppalaga thanks, it was much easier to create 1.5.8 now.

Just one more whish: Could you maybe use shorter commit/PR titles to avoid `...` in the PR title? Example: #631
It looks weird in the changelog, so had to copy the remaining text from the PR description to replace the `...`.

Thanks!

**Reply by ppalaga** [reply]
> Could you maybe use shorter commit/PR titles to avoid ... in the PR title? 

Oh man, if you knew how I hate GitHub for cutting my carefully crafted prose :-)
I'll try to.

And thanks for creating the releases!

@gastaldi any progress with jreleaser or whatever you wanted to introduce to automate the creation of GH releases?

**Reply by gastaldi** [reply]
@ppalaga haven't had time to finish that yet. 

Right now I create GH Releases manually, asking to generate the release notes through the GitHub UI.

**Reply by ppalaga** [reply]
> haven't had time to finish that yet.

I am staying tuned!

**Comment by famod** [comment]
@ppalaga something else I noticed about the 1.5 branch: it doesn't trigger any job on push/merge. Is this intentional?

**Reply by gastaldi** [reply]
I think this branch needs to be added here: https://github.com/quarkiverse/quarkus-cxf/blob/1.5/.github/workflows/build.yml#L6

**Reply by famod** [reply]
Yes, but before adding it wanted to make sure I'm not missing anything.

**Reply by ppalaga** [reply]
I was not aware of that and I must say, I was not really missing those jobs being triggered. Feel free to enable them if you think they are needed.

**Reply by famod** [reply]
Well, like `main`, `1.5` is a branch we release from and so it should "protected" with build on each change, no?
